playbooks/robusta_playbooks/k8s_resource_enrichments.py

```diff
@@ -135,3 +136,44 @@ def get_resource_yaml(event: KubernetesResourceEvent):
     except Exception as exc:
         logging.error("Unexpected error occurred!")
         logging.exception(exc)
+
+
+class StatusEnricherParams(ActionParams):
+    """
+    :var show_message: shows the message attached to each condition
+
+    """
+
+    show_details: bool = False
+
+
+@action
+def status_enricher(event: KubernetesResourceEvent, params: StatusEnricherParams):
+    """
+    Enrich the finding with the k8s objects's status conditions.
+
+    """
+    resource = event.get_resource()
+    if not resource:
+        logging.error(f"status_enricher was called on event without a resource : {event}")
+        return
+    if not resource.status.conditions:
+        event.add_enrichment(
+            [MarkdownBlock(f"*No status/conditions to report for {resource.kind} {resource.metadata.name}*")]
+        )
+    headers = ["Type", "Status"]
+    if params.show_details:
```

Comment:
Why do we need `show_details`?
Why not always show the message?

Reply:
I saw we didn't always show the message for all of our enrichers (like for `node_status_enricher`) 
So I wanted to make it optional to be able to replace all of the status enrichers with this generic one.

Reply:
For nodes its repetative
<img width="1123" alt="Screen Shot 2023-04-13 at 22 39 49" src="https://user-images.githubusercontent.com/97387909/231865748-dddb551a-2f38-485e-87f9-e932aac7e278.png">